examples/spi/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(spi)
+
+# root
+set(XMOS_SANDBOX_DIR ${CMAKE_SOURCE_DIR}/../../..)
+
+# target: choose the target platform
+set(APP_HW_TARGET src/XK_VOICE_L71.xn)
+# set(APP_HW_TARGET src/XK-EVK-XU316-AIV.xn)
```

Comment:
replace by XK-EVK-XU316-AIV.xn, check tile

Reply:
I implemented the suggested changes in https://github.com/xmos/lib_device_control/pull/109/commits/6e4ff5958c57934974294c98f079ee92cb4d57b5. They have not been tested on the Vision board, but only on the Voice board.